src/reducers/index.js

```diff
@@ -13,9 +13,18 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+    return {
+      ...state,
+      count: initialState.count++
```

Comment:
you are not thinking imutably. What you are doing is incrementing the count inside of the object you defined on line 3. initial state only exists to set the default redux store. after that, what you are passing in is the state variable. You don't want to return the object where you increment a count of a different object. You want to return the state you pass in where you add one to the count.

Reply:
Thanks Ryan, just updated and its working without requiring that extra click (no longer mutating initial state, just state)